src/apolo_app_types/outputs/update_outputs.py

```diff
@@ -55,6 +55,11 @@ async def update_app_outputs(helm_outputs: dict[str, t.Any]) -> bool:  # noqa: C
                 conv_outputs = await get_custom_deployment_outputs(helm_outputs)
             case AppType.SparkJob:
                 conv_outputs = await get_spark_job_outputs(helm_outputs)
+            case AppType.TextEmbeddingsInference:
+                labels = {"application": "tei"}
+                conv_outputs = await get_custom_deployment_outputs(
```

Comment:
let's maybe add dedicated output types, based on which we will integrate?

Reply:
The Custom Deployment Output is External and Internal URLs based on which we can integrate, there's no specific need to introduce a new type, 
Or, you mean, to distinguish somehow from which app is it?